src/main/webapp/js/studentProfile.js

```diff
@@ -36,6 +34,22 @@ $(function() {
             $('#profilePicEditRotateRight').click(function() {
                  picture.guillotine('rotateRight');
             });
+            $('#profilePicEditPanUp').click(function() {
+                var data = picture.guillotine('getData');
+                picture.guillotine('instance')._offset(data.x / data.w, (data.y - 10) / data.h);
```

Comment:
Nope, this is the recommended way by the library's author. They discussed if this functionality should be baked in to the library, but decided against it as this method was simple enough and they wanted to keep the library lean.
https://github.com/matiasgagliano/guillotine/issues/6